src/main/frege/frege/nativegen/Main.fr

```diff
@@ -170,14 +175,14 @@ parseArgs args = go optDefault args where
   go optSoFar ("-p" : packageName : rest)
     = go optSoFar.{name = packageName, category = Package} rest
   go optSoFar ("-c" : className : rest)
-    = go optSoFar.{name = className, category = Class} rest
+    = go optSoFar.{name = optSoFar.name ++ (if optSoFar.name == "" then "" else ",") ++ className, category = Class} rest
   go optSoFar ("-d" : directory : rest)
     = go optSoFar.{outputDir = directory} rest
   go optSoFar ("-t" : typesFilePath : rest)
     = go optSoFar.{typesFilePath = typesFilePath} rest
   go _ (invalidOption:_) = Left $ "Invalid option: " ++ invalidOption
 
-data Category = Package | Class
+data Category = Package | Class | Types
```

Comment:
I would rather remove the category `Types` and just use `Class` and keep the option as `-c class1,class2,class3`